### PR TITLE
Installed Mongodb, dotenv, cors. Set up serverdb.js and conn,js

### PR DESCRIPTION
**Pull request by jdustin2**
I installed Mongodb, dotenv, cors. Set up serverdb.js and conn,js to connect to the database.
Closes: #3

### CODE REVIEW

File: backend/server/serverdb.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const cors = require("cors");
+require("dotenv").config({ path: "./config.env" });
+const port = process.env.PORT || 5000;
+app.use(cors());
+app.use(express.json());
+app.use(require("./routes/record"));
+// get driver connection
+const dbo = require("./db/conn");
+ 
+app.listen(port, () => {
```
Comment (stoltenbergnathan) [inline backend/server/serverdb.js:12]: Should this be in server.js

Comment (jdustin2) [reply backend/server/serverdb.js:12]: sure, should we merge the files and put it into the server folder?

File: backend/server/serverdb.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+require("dotenv").config({ path: "./config.env" });
+const port = process.env.PORT || 5000;
```
Comment (stoltenbergnathan) [inline backend/server/serverdb.js:5]: This should be port 80

File: backend/server/conn.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const { MongoClient } = require("mongodb");
+const Database = process.env.ATLAS_URI;
+const client = new MongoClient(Database, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+ 
+var _db;
```
Comment (stoltenbergnathan) [inline backend/server/conn.js:8]: No var

File: backend/server/conn.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+module.exports = {
+  connectToServer: function (callback) {
+    client.connect(function (err, db) {
+      // Verify we got a good "db" object
+      if (db)
+      {
+        _db = db.db("TimeSync DataBase");
+        console.log("Successfully connected to MongoDB."); 
+      }
+      return callback(err);
```
Comment (stoltenbergnathan) [inline backend/server/conn.js:19]: console.log this error for easy bug fix

**Review by stoltenbergnathan — APPROVED** [review]

**Pull request merged**